Makefile

```diff
@@ -27,3 +28,8 @@ kube-update: update-manifests
 
 update-manifests:
 	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-logger-fluentd-daemon.yaml > manifests/deis-logger-fluentd-daemon.tmp.yaml
+
+build-plugin:
+
+	rake --rakefile=deis-output/Rakefile build
+	cp deis-output/pkg/*.gem rootfs/
```

Comment:
So it looks like we need Ruby to build this?  Can we containerize this? I love Ruby, but I hate having to load up my laptop with more and more dev tools.